Makefile

```diff
@@ -77,6 +77,9 @@ HAS_DEP          := $(shell $(CHECK) dep)
 HAS_GOLANGCI     := $(shell $(CHECK) golangci-lint)
 HAS_GOIMPORTS    := $(shell $(CHECK) goimports)
 GOLANGCI_VERSION := v1.16.0
+HAS_COCOV_XML := $(shell command -v gocov-xml;)
```

Comment:
Ugh, this is a typo.

Reply:
lmk when it's fixed and u want a review!